static/openapi/eds-spec.json

```diff
@@ -0,0 +1,377 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "PhotonIQ Global Event Delivery Service (EDS)",
+    "description": "The PhotonIQ Global Event Delivery Service (EDS) is a scalable platform for delivering real-time event data to your applications. It has being designed and implemented for scaling to thousands of concurrent connections. EDS enables you to seamlessly integrate event-driven functionality into your systems, ensuring timely updates and improved responsiveness.\n\nFeatures:\n- Real-time event delivery via WebSockets with one level de-duplication of events\n- Subscribe to events\n- Publish to event stream\n- Metrics gathering on connections\n- Health and version api check for service\n\nExplore the API documentation below to discover the available endpoints, request and response formats, and how to integrate EDS into your projects.\n",
+    "version": "1.0.0"
+  },
+  "tags": [
+    {
+      "name": "Event Publish",
+      "description": "APIs related to event publishing."
+    },
+    {
+      "name": "Event Subscription",
+      "description": "APIs related to event subscriptions and streaming."
+    },
+    {
+      "name": "EDS Metrics",
+      "description": "APIs for retrieving metrics data related to EDS."
+    },
+    {
+      "name": "EDS Status",
+      "description": "APIs for checking the health and version of EDS."
+    }
+  ],
+  "paths": {
+    "/fabric/{fabricName}/api/es/v1/stream/{streamName}/publish": {
```

Comment:
This does not follow the photoniq guidelines.

All apis should start with /api

Reply:
Done, also changed in code for the same